packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts

```diff
@@ -334,8 +333,9 @@ export class CucumberJSAllureFormatter extends Formatter {
       return;
     }
 
-    const { fileName = "attachment", body, mediaType } = data;
-    const attachmentFilename = this.allureRuntime.writeAttachment(body, mediaType);
+    const { fileName = "attachment", body, mediaType, contentEncoding } = data;
+    const encoding = Buffer.isEncoding(contentEncoding) ? contentEncoding : undefined;  // only pass through valid encodings
```

Comment:
Updated usage of the function to check for a valid encoding.  If valid, it passes it through.  Invalid changes it to undefined for the system to ignore later.